### DISCUSSION TITLE
Updated Report Categories

### DISCUSSION THREAD

**Discussion by JorgeRivera94** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#287] [Ideas]
Hi! This post follows up on the proposed changes in #173 and is a result of issue #177.

TL;DR at the end.

I evaluated UPRM's Criminal Acts Reports from January 2023 to August 2024 to ensure that our report categories are relevant and provide a quick way for users to report crimes or alerts, as ensuring the process is as quick and simple as posible, helps users prioritize their safety.

These are the definitions developed to communicate and study these reports:
* **Official Type**: Type of criminal act as assigned by the official report.
* **Victim Type**: Type of complainant as assigned by the official report.
* **Actor Type**: Type of criminal actor as described in the report.
* **Severity**: Integer value on the scale [1,5] assigned by me. Priority was be given to physical security; meaning 1 represents the lowest physical safety risk, and 5 represents high physical safety risk.

Each report was assigned a **Severity** score based on the report description and how much of a threat it presented to the complainant.

Ultimately, **Victim Type** and **Actor Type** did not seem relevant to report categories when taking into consideration official reports and conversations with employees. 

For the new categories, I wanted to focus on the types of categories (**Official Type**) with an averaged severity of 3 or greater.

An interesting observation is that even though the illegal appropriation of public property was the most reported crime (31.7%), it's averaged severity score had a value of 1.42 and therefore does not present a significant safety risk for our application to consider in the report creation.

The plots bellow describe the data in terms of **Official Type**, and **Severity**.
![Pareto Plot of Official Type of Crimes](https://github.com/user-attachments/assets/b10e7472-39b3-4581-9254-8c3f58a88b03)
![Averaged Severity of official Type of Crimes](https://github.com/user-attachments/assets/bf623f84-822f-4780-b739-8cc87993d139)

The current report categories are: "Safety Issue", "Suspicious Person", "Armed Suspect" and "Sexual Harasser".

We see that the **Official Types** with the highest Averaged Severity are Cyber-Harrassment, Harassment, Stalking, Threat, and Aggravated Assault with scores higher than 4. Followed by Assault, Dishonest Exposure and Disruption to the Peace with scores higher than or equal to 3.

Having identified the 8 most severe types of crimes in UPRM campus helps limit our options for SafeRUM's report categories with relevant categories. However, providing 8 different categories to the user in a time sensitive situation can turn the report creation process into a hindrance for the user's safety.

Having read the reports, some of these categories can be aggregated due to the similarities in their descriptions and severity. This allows us to combine these **Official Types** with our current report categories.

Groups:
* Harassment: Contains **Official Types** Cyber-harassment, Harassment, Stalking.
* Threat / Assault: Contains **Official Types** Threat, Aggravated Assault, Assault, Disruption to the Peace and current labels "Armed Suspect" and "Suspicious Person".
* Sexual Harasser: Contains **Official Type** Dishonest Exposure and current label "Sexual Harasser".
* Safety Issue: Contains the current label "Safety Issue".

This way we can maintain our label "Safety Issue" and "Sexual Harasser", while replacing "Suspicious Person" and "Armed Suspect" labels with "Harassment" and "Threat / Assault".

TL;DR
Report creation labels should be:
* Harassment
* Threat / Assault
* Sexual Harasser
* Safety Issue

**Comment by BEINGCYBERSTALKEDHELP** [comment]
Could you help me please! I’m dealing with a group of people who’ve hacked into my phone and stripped me of everything I own. I am to believe it’s a huge criminal organization that has compromised my carrier… because why is it that all locations clearly see my phone not acting normal and my device getting so hot it burns anyone’s hand. I cannot feed myself, for months and I’m led to believe what I see is normal, but my entire family for some reason was turned against me (I live alone across the country) so I’m an easy target. I have no idea what to do, but Apple has verified that everything I’m saying is true. Though no one is helping me as if it’s not urgent. I believe my life is on the line , the hackers are definitely trying to make me believe I’ve got some loose screws. I probably do, due the type of men I choose to have a “relationship” with. Though I’ve learned my lesson and just want to be able to get some money go back home and spend time with my little sister- my family. My parents both are battling Cancer and my siblings only have me. I want to help them feel secured and be that brother they can depend on. I’m so tired of this abuse, now I’m not dumb to commit suicide, I’m all good. I don’t mind the fact they stole $20k+ I just want to be able to be their for my family and help people really I do. Now I don’t even get that and probably will have to live with the traumatic experience.
